Nautilus/Handlers/ModDataBankHandler.cs

```diff
@@ -0,0 +1,111 @@
+using BepInEx;
+using Nautilus.Utility;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Nautilus.Handlers;
+/// <summary>
+/// A handler class for adding databank entries for mods.
+/// </summary>
+public static class ModDatabankHandler
+{
+    private static bool isinit;
+    private static List<PDAEncyclopedia.EntryData> waitlist = new();
+    internal static bool isEnabled = true;
+    internal static void Initialize(uGUI_EncyclopediaTab tab)
+    {
+        LanguageHandler.SetLanguageLine("EncyPath_Mods", "Mods");
+        isinit = true;
+        foreach (var data in waitlist)
+        {
+            CompleteRegister(data);
+        }
+    }
+    private static void CompleteRegister(PDAEncyclopedia.EntryData data)
+    {
+        if (isinit && isEnabled)
+        {
+            InternalLogger.Info($"{data.key} entry added.");
+            PDAHandler.AddEncyclopediaEntry(data);
+#if BZ_STABLE
+            PDAEncyclopedia.Add(data.key, true, false);
+#else
+            PDAEncyclopedia.Add(data.key, true);
+#endif
+        }
+        else
+        {
+            waitlist.Add(data);
+        }
+    }
+    /// <summary>
+    /// Register mod with database using default values. 
```

Comment:
I still could not tell you what this method does based on the "Register mod with database" documentation, without looking into the source code. The same goes with the other method.

Reply:
Pretend the modder knows nothing about this method and write your documentation like that.

Reply:
It's not like you need to list everything it does internally, but the user should know all of its effects and its purpose.